target/linux/bcm63xx/base-files/lib/upgrade/platform.sh

```diff
@@ -1,9 +1,19 @@
 PART_NAME=linux
+REQUIRE_IMAGE_METADATA=0
+
 platform_check_image() {
 	[ "$#" -gt 1 ] && return 1
+
+	case "$(board_name)" in
+		comtrend,vr-3032u)
+			# NAND sysupgrade
+			return 0
+			;;
+	esac
+
 	case "$(get_magic_word "$1")" in
 		3600|3700|3800)
-			# 6348GW-11 boards use openwrt-96348GW-11-squashfs-cfe.bin files
+			# CFE tag versions
```

Comment:
OT: Do you have any idea how many boards are covered here? If it's not too painful, I'd consider switching to board_name for them as well, so we end up with only one case?

Reply:
Basically every bcm63xx board is covered here, since all of them rely on the CFE Tag.